### ISSUE TITLE
UI Improvements/Anzeigefehler beim Bierruf-Ort Android

### ISSUE THREAD

**Issue by jaronhsv** [r0bs/bn#76]
Bei meiner Android-APK Version wird der Ort des Bierrufes wenn er zu lang ist nicht mit [...] am Ende angezeigt, sondern geht wie auf dem Screenshot zu sehen über den Bildrand hinaus.

Außerdem könnte Datum/Wochentag und Uhrzeit weiter in die obere rechte Ecke des Bierrufes gerückt werden, evtl. auf Höhe "... trinkt Bier!". (Kein Android spezifisches UI improvement; auch bei IOS)

![Screenshot_20241011-114202](https://github.com/user-attachments/assets/f665c60e-3893-4776-a3e0-e65dbc097229)

**Comment by r0bs** [comment]
@jaronhsv, herzlich willkommen und danke für die guten Vorschläge! Kannst du mal schauen, ob in der aktuellen Version [0] lange Ortsnamen bereits ellipsiert dargestellt werden?

[0] https://www.dropbox.com/scl/fi/wcmo8s6aj9js6stefxslk/223.apk?rlkey=ad7fr009x28g6kks9vhaxjmib&st=ha1viojw&dl=0

**Comment by jaronhsv** [comment]
Ja, Ortsnamen werden jetzt ellipsiert dargestellt. Lag vielleicht an der illegalen APK?

**Comment by r0bs** [comment]
Ja, die entsprach wohl nicht mehr ganz dem aktuellen Stand. Eine
Optimierung der Datumsposition folgt demnächst!
